### PR TITLE
Feature change pipenv for poetry dependency manager

### PR DESCRIPTION
**Pull request by kbeker**
There is no issue for that.

### CODE REVIEW

**Comment by Jakub89** [comment]
@kbeker We would need to change README.md also with that PR.
```
2. Install pipenv on your machine. Do not create any virtualenv, just install pipenv globally on your operation system:

Arch:
$ sudo pacman -Syu python-pipenv
Debian:
$ sudo apt install pipenv

3. Create new virtual environment for project with all development packages.
$ pipenv install --dev

4. Activate newly created virtualenv:

$ pipenv shell
```

File: install-environment-packages.sh [hunk 1]
```diff
@@ -6,14 +6,14 @@ export PIPENV_VERBOSITY=-1
 # Run pipenv virtual environment
```
Comment (Jakub89) [inline install-environment-packages.sh]: Old comments. Do we even need `PIPENV_VERBOSITY` variable?

File: install-environment-packages.sh [hunk 2]
```diff
@@ -24,9 +24,7 @@ fi
 
 (
     cd ${BASH_SOURCE%/*}/
-    pipenv uninstall --all
-    pipenv install --dev --clear
-    pipenv check
+    poetry install
 )
 
 # Disable pipenv virtual environment
```
Comment (Jakub89) [inline install-environment-packages.sh]: Old comment.

**Comment by Jakub89** [comment]
`run-test-coverage.sh` need fixing too.

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "project-liberation"
+version = "v1.10.0"
+description = "Necessary packages for Project Liberation"
+authors = ["Karol Beker <karol.beker@codepoets.it>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
```
Comment (Jakub89) [inline pyproject.toml]: Let's change it to `python = "3.7.*"` like we had to on my machine, so it's always use newest 3.7 version.

Comment (kbeker) [reply pyproject.toml]: Done

File: install-environment-packages.sh (hunk 1 above)
Comment (kbeker) [reply install-environment-packages.sh]: Fixed all old comments and delete unused pipenv variables

File: install-environment-packages.sh (hunk 2 above)
Comment (kbeker) [reply install-environment-packages.sh]: Done

**Comment by kbeker** [comment]
I updated everything @Jakub89 so I'm merging it right now:)

**Pull request merged**